jvb/src/main/kotlin/org/jitsi/videobridge/stats/config/StatsManagerBundleActivatorConfig.kt

```diff
@@ -16,141 +16,97 @@
-                        transports as ConfigList
-                        transports.map { it as ConfigObject }
+                }
+            "videobridge.stats"
+                .from(JitsiConfig.newConfig)
+                .convertFrom<ConfigObject> { cfg ->
+                    val transports = cfg["transports"]
+                        ?: throw ConfigException.UnableToRetrieve.NotFound("Could not find transports within stats")
+                    transports as ConfigList
+                    transports.map { it as ConfigObject }
                             .map { it.toConfig() }
-                            .mapNotNull { StatsTransportConfig.fromNewConfig(it) }
-                    }
+                            .mapNotNull { it.toStatsTransportConfig() }
                 }
-            )
+        }
+    }
 
-            /**
-             * Note: These three classes exist only for the purposes of validation
-             * and future deprecation.  IN the legacy config file, the properties
-             * for stats transports are spread across these 3 values, making it
-             * difficult to encompass in a single property.  We could read the property
-             * at the level of 'org.jitsi.videobridge", but that has the following
-             * issues:
-             *
-             * 1) It won't register correctly in validation that these specific properties
-             * are read
-             * 2) When we want to mark them as deprecated, we'd be marking the entire chunk
-             * as deprecated instead of the specific properties
-             *
-             * Because of that, we define these here to handle the above use cases, but
-             * implement the property that's used differently (see above)
-             */
-            @Suppress("unused")
-            private class LegacyStatsTransportsProperty : SimpleProperty<String>(
-                legacyConfigAttributes {
-                    name("org.jitsi.videobridge.STATISTICS_TRANSPORT")
-                    readOnce()
-                }
-            )
+    /**
+     * From a map of properties pulled from the legacy config file, create a list of [StatsTransportConfig]
+     */
+    private fun Map<String, String>.toStatsTransportConfig(): List<StatsTransportConfig> {
+        val transportTypes =
+            this["org.jitsi.videobridge.STATISTICS_TRANSPORT"]?.split(",") ?: return listOf()
+        return transportTypes.mapNotNull { transportType ->
+            val interval = this["org.jitsi.videobridge.STATISTICS_INTERVAL.$transportType"]?.let {
+                Duration.ofMillis(it.toLong())
+            } ?: this@StatsManagerBundleActivatorConfig.interval
+            when (transportType) {
+                "muc" -> StatsTransportConfig.MucStatsTransportConfig(interval)
+                "callstats.io" -> StatsTransportConfig.CallStatsIoStatsTransportConfig(interval)
```

Comment:
You use "callstats.io" here and "callstatsio" below. Is it intentional?

Reply:
Looks like this is how it was in master...don't know why I changed it to `callstatsio` in new config but looks like I did.  I may have considered using the name as 'keys' at one point within an object, which would mean they couldn't have dots. 🤷‍♂️